docs/modules/ROOT/pages/how-to/use-tilt.adoc

```diff
@@ -0,0 +1,201 @@
+= How to use Tilt with {product}
+
+image:logos/tilt.svg[role="related thumb right",alt="Tilt logo",width=120,height=120] https://tilt.dev/[Tilt] is a developer toolkit to enable teams to build microservices architectures faster on Kubernetes clusters.
+
+> The best indicator of a healthy development workflow is a short feedback loop.
+
+https://docs.tilt.dev/example_go.html[(Source: Tilt documentation)]
+
+This page will describe the required steps to use Tilt successfully with {product}.
+
+IMPORTANT: This information is provided as a courtesy for {product} users. We don't provide support for Tilt, and in case of issues, you are invited to contact the developers of Tilt at their https://github.com/tilt-dev/tilt[GitHub project].
+
+== Requirements
+
+To follow this guide, please make sure that you have the following tools installed:
+
+`tilt`:: The https://docs.tilt.dev/[Tilt documentation] explains how to install it in your operating system of choice.
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry.
+
+`docker`:: You can download it from https://www.docker.com.
+
+== Procedure
+
+This section explains the steps required to use Tilt with {product}.
+
+=== 1. Login to {product}
+
+Follow these steps to login to {product} on your terminal and to create a project:
+
+. Login to the {product} console:
++
+[source,shell]
+--
+$ oc login --server=https://api.[YOUR_CHOSEN_ZONE].appuio.cloud:6443
+You must obtain an API token by visiting https://oauth-openshift.apps.[YOUR_CHOSEN_ZONE].appuio.cloud/oauth/token/request
+--
+
+. Click on the link above to open it on your browser.
+. Click "Display token" and copy the login command shown in "Log in with this token"
+. Paste the `oc login` command on the terminal:
++
+[source,shell]
+--
+$ oc login --token=sha256~_xxxxxx_xxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxx-X --server=https://api.[YOUR_CHOSEN_ZONE].appuio.cloud:6443
+--
+
+. Create a new project called "[YOUR_USERNAME]-fortune-go"
++
+[source,shell]
+--
+$ oc new-project [YOUR_USERNAME]-fortune-go
+Now using project "[YOUR_USERNAME]-fortune-go" on server "https://api.[YOUR_CHOSEN_ZONE].appuio.cloud:6443".
+--
+
+=== 2. Login to the {product} Container Registry
+
+Tilt uses Docker to build the container image, and to publish to a container registry. In this example we're going to use the {product} integrated container registry.
+
+. Login to the {product} container registry (as explained in xref:how-to/use-integrated-registry.adoc[this page]):
++
+[source,shell]
+--
+$ oc whoami -t | docker login registry.${zone}.appuio.cloud -u $(oc whoami) --password-stdin
+--
+
+=== 3. Clone the sample "Fortune in Go" project
```

Comment:
Now this becomes the character of a tutorial, isn't it?

Reply:
Well, according to Divio "How To" guides are problem oriented rather than learning oriented, and here we're solving a specific problem: using Tilt with APPUiO, which is not trivial (nor covered in the Tilt docs, for that matter)